Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/AWSCredentialIdentityResolvers/SSOAWSCredentialIdentityResolver.swift

```diff
@@ -62,14 +79,18 @@ public struct SSOAWSCredentialIdentityResolver: AWSCredentialIdentityResolver {
 
         var ssoToken: BearerTokenIdentity!
         var region: String!
+        var tokenFeatureIDs: [String]!
         if let ssoSessionName {
             region = try getProperty(ssoSessionName, .ssoSession, "sso_region", fileBasedConfig)
+            tokenFeatureIDs = [CredentialFeatureID.CREDENTIALS_PROFILE_SSO.rawValue]
             ssoToken = try await SSOBearerTokenIdentityResolver(
                 profileName: resolvedProfileName,
-                configFilePath: configFilePath
+                configFilePath: configFilePath,
+                credentialFeatureIDs: credentialFeatureIDs + tokenFeatureIDs
```

Comment:
Call `InternalSSOOIDC::createToken` with feature IDs resolved so far. The call is made within `SSOBearerTokenIdentityResolver` if token needs to be refreshed.